src/schema/workflow_execution_activity.yaml

```diff
@@ -198,11 +199,14 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:wfmgan-{id_shoulder}-{id_blade}{id_version}{id_locus}"
           interpolated: true
+      img_identifiers:
+        multivalued: false
```

Comment:
if `img_identifiers` is going not to be `multivalued`, that should be asserted in it's global definition, not in multiple `slot_usage`s. We shouldn't have any slots that are `multivalued` in some classes but not in others.

It's parent slot, `external_database_identifiers` is `multivalued`. If you want to constrain `img_identifiers` to only allow a single value, we should leave it as a multivalued list and set [maximum_cardinality](https://linkml.io/linkml-model/latest/docs/maximum_cardinality/) to `1`